libs/langgraph/src/pregel/call.ts

```diff
@@ -0,0 +1,63 @@
+import { RunnableConfig, RunnableSequence } from "@langchain/core/runnables";
+import { AsyncLocalStorageProviderSingleton } from "@langchain/core/singletons";
+import { CONFIG_KEY_CALL, RETURN, TAG_HIDDEN } from "../constants.js";
+import { ChannelWrite, PASSTHROUGH } from "./write.js";
+import { RetryPolicy } from "./utils/index.js";
+import { Promisified } from "./types.js";
+import { RunnableCallable, type RunnableCallableArgs } from "../utils.js";
+/**
+ * Cache for storing runnable sequences to avoid recreating them
+ */
+
+/**
+ * Get a runnable sequence for a function that wraps it in a sequence with a RETURN channel write
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function getRunnableForFunc<FuncT extends (...args: any[]) => any>(
+  name: string,
+  func: FuncT
+): RunnableSequence<Parameters<FuncT>, ReturnType<FuncT>> {
+  const run = new RunnableCallable<Parameters<FuncT>, ReturnType<FuncT>>({
+    func: (input: Parameters<FuncT>) => func(...input),
+    name,
+    trace: false,
+    recurse: true,
```

Comment:
this actually shouldn't recurse for task/entrypoint, I'll change it in py if it was accidentally set to do so

Reply:
I think it was, but I could've misread - good eye, either way

Reply:
yea opened pr in py